icu4j/main/tests/core/src/com/ibm/icu/dev/test/impl/UnitsTest.java

```diff
@@ -141,7 +141,7 @@ void testATestCase(ComplexUnitsConverter converter) {
             new TestCase("light-year", "light-year-and-meter", BigDecimal.valueOf(1.0 + 1e-15),
                          new Measure[] {new Measure(1, MeasureUnit.LIGHT_YEAR),
                                         new Measure(9.46073, MeasureUnit.METER)},
-                         0 /* meters, precision */),
+                         0.0000005 /* meters, precision */), // TODO check, with CLDR 42 conversions get 9.460730472580800
```

Comment:
The new value is more accurate, because the IAU uses the Julian year. So the test needs to be fixed.

https://www.iau.org/public/themes/measuring/#:~:text=The%20light%2Dyear%20is%20roughly,9%2C460%2C730%2C472%2C580.8%20km%20or%2063%2C241%20au.

To think of it in easily accessible terms, the light-year is 9,460,730,472,580.8 km

Reply:
OK thanks, I will fix the test